contracts/IRevoBounty.sol

```diff
@@ -13,6 +13,11 @@ interface IRevoBounty {
         view
         returns (TokenAmount[] memory);
 
+    function calculateAdditionalBountyFee(TokenAmount[] calldata interestAccrued)
```

Comment:
In my mind, a "bounty fee" is a fee the user pays that turns into a bounty for the person who calls `claimRewards` to collect. By that logic though, "additional bounty fee" doesn't make sense. The "additional bounty" is not funded by fees. It's a bounty that Revo pays for, presumably out of the community fund (which will have governance tokens, withdrawal fees, etc)

Reply:
I think `calculateAdditionalBounty` would be easier to understand